Input/InputDecoratorIcon.js

```diff
@@ -47,7 +47,7 @@ const InputDecoratorIconBase = kind({
 	},
 
 	defaultProps: {
-		size: 'large'
+		size: 'small'
```

Comment:
If changing it, it will affect other skins. 
In my opinion, if you want to use the `small` in the `silicon` skin,
wrap `InputDecoratorIconBase` with `Skinnable`
and the size default prop could be removed,
then we could add the `size` in a `computed`.

```
	computed: {
		className: ({position, styler}) => {
			return styler.append('icon' + (position === 'before' ? 'Before' : 'After'));
		},
		size: ({skin}) => (skin === 'silicon' ? 'small' : 'large')
	},
```

How about this approach?